pkg/jobrunaggregator/jobrunaggregatorlib/ci_data_client.go

```diff
@@ -347,7 +347,7 @@ func buildMasterNodesUpdatedSQL(masterNodesUpdated string) string {
 	masterNodesUpdatedSQL := ""
 
 	if len(masterNodesUpdated) > 0 {
-		masterNodesUpdatedSQL = fmt.Sprintf("AND MasterNodesUpdated = '%s'", masterNodesUpdated)
+		masterNodesUpdatedSQL = fmt.Sprintf("AND JobRuns.MasterNodesUpdated = '%s'", masterNodesUpdated)
```

Comment:
Future consideration, would we ever need to pass in the table name so it didn't always have to be JobRuns?

Reply:
I actually hope to kill all the JobRuns tables, so the column shouldn't be permanently ambiguous. We could also just not alias the one that was aliased, we're using the same table below technically.